interfaces/cython/cantera/examples/reactors/ic_engine.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+"""
+Simulation of a (gaseous) Diesel internal combustion engine.
+
+The reaction mechanism used is the Madison n-heptane reaction mechansim. It
+can be downloaded from https://www.erc.wisc.edu/chemicalreaction.php
+(Caution: use uppercase species names!)
+
+@author: Thomas Fiala, Lehrstuhl für Thermodynamik, TU München
```

Comment:
If you don't mind, I would prefer to keep attributions in (a) the AUTHORS file and (b) the Git commit log.